rudder-agent/SOURCES/configure

```diff
@@ -225,11 +225,11 @@ test_cc() {
 # test options one by one to enable as many as possible
 
 # try strong first
-test_cc "SECURE" "-fstack-protector-strong" ""
+test_cc "SECURE" "-fstack-protector-strong" "-fstack-protector-strong"
```

Comment:
why passing it as ldflag?

Reply:
because security protections need various libraries and the linker knows which ones but sometimes doesn't guess by itself it should include them